bin/update-module-translations.mjs

```diff
@@ -0,0 +1,32 @@
+/**
+ * This script is used for downloading the latest Zendesk official translation files for a module in the `src/module` folder.
+ * 
+ * Usage: node bin/update-module-translations.mjs [PACKAGE_NAME] [OUTPUT_PATH]
+ * Example: node bin/update-module-translations.mjs new-request-form src/modules/new-request-form/translations/locales
+ */
+import { writeFile } from "node:fs/promises";
+import { resolve } from "node:path";
+
+const PACKAGE_NAME = process.argv[2];
+const OUTPUT_PATH = resolve(process.cwd(), process.argv[3]);
```

Comment:
I'm also thinking that maybe we should not let the output path be configurable.
If it's set by default than we ensure that all apps are consistent when it comes to storing their locales.
One more argument for it is that the rollup config seems to expect translations to be in `src/modules/MODULE_NAME/translations/locales/*.json` so we really should not break away from it.